flows/vision-zero/make_kv_store.py

```diff
@@ -17,6 +17,14 @@
     "AWS_CSV_ARCHIVE_BUCKET_NAME": os.getenv("AWS_CSV_ARCHIVE_BUCKET_NAME"),
     "AWS_CSV_ARCHIVE_PATH_PRODUCTION": os.getenv("AWS_CSV_ARCHIVE_PATH_PRODUCTION"),
     "AWS_CSV_ARCHIVE_PATH_STAGING": os.getenv("AWS_CSV_ARCHIVE_PATH_STAGING"),
+    "PRODUCTION_GRAPHQL_ENDPOINT": os.getenv("PRODUCTION_GRAPHQL_ENDPOINT"),
+    "PRODUCTION_GRAPHQL_ENDPOINT_ACCESS_KEY": os.getenv("PRODUCTION_GRAPHQL_ENDPOINT_ACCESS_KEY"),
+    "OCR_CR3_SOURCE_BUCKET": os.getenv("OCR_CR3_SOURCE_BUCKET"),
+    "OCR_CR3_SOURCE_PATH": os.getenv("OCR_CR3_SOURCE_PATH"),
+    "OCR_DIAGRAM_TARGET_BUCKET": os.getenv("OCR_DIAGRAM_TARGET_BUCKET"),
+    "OCR_DIAGRAM_TARGET_PATH": os.getenv("OCR_DIAGRAM_TARGET_PATH"),
+    "OCR_BATCH_SIZE": os.getenv("OCR_BATCH_SIZE"),
+    "OCR_SINGLE_CRASH": os.getenv("OCR_SINGLE_CRASH"),
```

Comment:
Bringing more configuration values out of the Airflow JSON blob and into the top level JSON blob as found in the prefect KV store.